conf/synapse-coturn.service

```diff
@@ -8,7 +8,7 @@ User=turnserver
 Group=turnserver
 Type=notify
 EnvironmentFile=/etc/matrix-__APP__/coturn_env
-ExecStart=/usr/bin/turnserver -c /etc/matrix-__APP__/coturn.conf $EXTRA_OPTIONS --pidfile=
+ExecStart=/usr/bin/turnserver -c /etc/matrix-__APP__/coturn.conf $EXTRA_OPTIONS -L 0.0.0.0 --pidfile=
```

Comment:
is it really needed to bind to any public addresses?
this is not reverse proxied?

Reply:
Does not look like this is proxied. I do not find any reference to the port that is used (5351 in my case) in nginx config and the port is open on the firewall.